src/Storage/Filters/ClientIpCheckActionFilterAttribute.cs

```diff
@@ -0,0 +1,46 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace Altinn.Platform.Storage.Filters
+{
+    /// <summary>
+    /// ClientIpCheckActionFilter
+    /// </summary>
+    public class ClientIpCheckActionFilterAttribute : ActionFilterAttribute
+    {
+        /// <summary>
+        /// List of valid ip addresses
+        /// </summary>
+        public string Safelist { get; set; }
+
+        /// <summary>
+        /// Authorize from ip address
+        /// </summary>
+        /// <param name="context">context</param>
+        public override void OnActionExecuting(ActionExecutingContext context)
+        {
+            string ipAddressList = context.HttpContext?.Request.Headers["X-Forwarded-For"].ToString();
+            bool validIp = true;
+            if (!string.IsNullOrEmpty(ipAddressList))
+            {
+                validIp = false;
+                foreach (string ipAddress in Safelist.Split(';'))
```

Comment:
Dette resulterer vel i at vi splitter listen for hver request som kommer inn. Noen måte å unngå dette på ?